newsletter/admin.py

```diff
@@ -43,13 +44,22 @@
 
 from .settings import newsletter_settings
 
+
+logger = logging.getLogger(__name__)
+
+
 # Contsruct URL's for icons
-ICON_URLS = {
-    'yes': '%snewsletter/admin/img/icon-yes.gif' % settings.STATIC_URL,
-    'wait': '%snewsletter/admin/img/waiting.gif' % settings.STATIC_URL,
-    'submit': '%snewsletter/admin/img/submitting.gif' % settings.STATIC_URL,
-    'no': '%snewsletter/admin/img/icon-no.gif' % settings.STATIC_URL
-}
+def icon_url(icon, alt_text, _urls={}):
+    if not _urls:
+        _urls['yes'] = static('newsletter/admin/img/icon-yes.gif')
+        _urls['wait'] = static('newsletter/admin/img/waiting.gif')
+        _urls['submit'] = static('newsletter/admin/img/submitting.gif')
+        _urls['no'] = static('newsletter/admin/img/icon-no.gif')
+
+    return format_html(
+        '<img src="{}" width="10" height="10" alt="{}"/>',
+        _urls[icon], alt_text
+    )
```

Comment:
Since SVGs are just a different file type, this should work fine. I can add a small docstring, I was actually going to move this into a mixin to add to the `ModelAdmin` so that it is easy for subclasses to override. I use my own icon font for admin stuff, so it'd be nice to easily be able to change the icon HTML from here as well to use my own icons.

The `_urls` parameter is a cheap way of caching the results of the `static` function. Since the dict is mutable and is a keyword arg, changes to it persist across calls to `icon_url`. The alternative would be to use `lru_cache` to cache the result of the function, but that caches on all arguments, so different alt text would lead to the same URL being cached multiple times (and `static` run each time) for the different alt text.